lib/standard/math.nit

```diff
@@ -285,6 +293,12 @@ redef class Float
 	#  * 1 if self is positive infinity
 	#  * -1 if self is negative infinity
 	#  * 0 otherwise
+	#
+	# ~~~
+	# assert 10.0.is_inf == 0
+	# assert inf.is_inf == 1
+	# assert (-inf).is_inf == -1
+	# ~~~
 	fun is_inf: Int do
```

Comment:
Now that I see this, was there any reason why this returns an `Int` and not a `Bool` ?